packages/style-engine/class-wp-style-engine.php

```diff
@@ -24,14 +24,16 @@
  */
 final class WP_Style_Engine {
 	/**
-	 * Style definitions that contain the instructions to
-	 * parse/output valid Gutenberg styles from a block's attributes.
+	 * Style definitions that contain the instructions to parse/output valid Gutenberg styles from a block's attributes.
 	 * For every style definition, the follow properties are valid:
 	 *  - classnames    => (array) an array of classnames to be returned for block styles. The key is a classname or pattern.
 	 *                    A value of `true` means the classname should be applied always. Otherwise, a valid CSS property (string)
 	 *                    to match the incoming value, e.g., "color" to match var:preset|color|somePresetSlug.
-	 *  - css_vars      => (array) an array of key value pairs used to generate CSS var values. The key is a CSS var pattern, whose `$slug` fragment will be replaced with a preset slug.
-	 *                    The value should be a valid CSS property (string) to match the incoming value, e.g., "color" to match var:preset|color|somePresetSlug.
+	 *  - css_vars      => (array) an array of key value pairs used to generate CSS var values.
+	 *                     The key should be the CSS property name that matches the second element of the preset string value,
+	 *                     i.e., "color" in var:preset|color|somePresetSlug. The value is a CSS var pattern,
+	 *                     whose `$slug` fragment will be replaced with the preset slug, which is the third element of the preset string value,
+	 *                     i.e., `somePresetSlug` in var:preset|color|somePresetSlug.
```

Comment:
Tiny nit: it's a fairly complex sentence this one, and I was wondering if it'd help giving an example of the CSS var pattern, so that it's clear what the intended value will be. Otherwise at first glance, you might think the value should be in the `var:` format.

```suggestion
	 *                     i.e., "color" in var:preset|color|somePresetSlug. The value is a CSS var pattern (e.g. `--wp--preset--color--$slug`),
	 *                     whose `$slug` fragment will be replaced with the preset slug, which is the third element of the preset string value,
	 *                     i.e., `somePresetSlug` in var:preset|color|somePresetSlug.
```

Reply:
But I was keeping it esoteric to maintain intellectual dominion in exclusion of all others! :trollface: 

Just kidding. It is a mouthful, thanks for trying to unweave it. 

If it's accurate for now I can come back and make it better.

Reply:
> If it's accurate for now I can come back and make it better.

Totally, not a blocker at all! We can be iterative about the wording 🙂